task2.js

```diff
@@ -0,0 +1,17 @@
+const csvFilePath = './csv/nodejs-hw1-ex1.csv'
+import csv from 'csvtojson'
+import fs from 'fs'
+
+const readStream = fs.createReadStream(csvFilePath)
+const writeStream = fs.createWriteStream('./out.txt', 'utf8')
+
+const handleError = () => {
+    readStream.destroy()
+    writeStream.end('Finished with error...')
```

Comment:
you should not log an error event into the file, please remove the message parameter from ```end()```.

you need to destroy ```csv()``` stream too.